book.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person, book)
+    @rentals << Rental.new(date, person, book)
+  end
+
+  def classroom=(classroom)
+    @classroom = classroom classroom.students.push(self) unless classroom.students.include?(self)
+  end
```

Comment:
- [ ] [**REQUIRED**] As requested by the previous reviewer, To implement the `belongs-to` association between `Classroom` and `Student`, a method that serves as the 'belongs-to' side (a student belongs to a classroom) should be present in the Student class.
- Kindly create a new file and write this code there, For ex:- 
- * When setting the classroom for a student it also adds it to the classrooms' students:

```ruby
class Student < Person
  attr_reader :classroom
  .
  .
  # *Check for a method that serves as the 'belongs-to' side (a student belongs to a classroom). Link to the article with examples: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```